test/test_PostRequestHandler.cpp

```diff
@@ -0,0 +1,80 @@
+#include "gmock/gmock.h"
+#include <gtest/gtest.h>
+
+#include <stdexcept>
+#include <string>
+
+#include "MockFileSystemPolicy.hpp"
+
+#include "PostRequestHandler.hpp"
+
+using ::testing::NiceMock;
+using ::testing::Return;
+
+class PostRequestHandlerTest : public ::testing::Test {
+protected:
+	PostRequestHandlerTest() {
+
+		fakeStat.st_size = 1024;
+		fakeStat.st_mtime = 777;
+
+		ON_CALL(fileSystemPolicy, writeToFile)
+		.WillByDefault(testing::Return());
```

Comment:
Since you are `using ::testing::Return;` on line 12 you can just generally just `Return()` without `testing::`

Reply:
Removed